includes/express-checkout/class-wc-payments-express-checkout-button-handler.php

```diff
@@ -446,4 +452,23 @@ public function filter_gateway_title( $title, $id ) {
 
 		return $title;
 	}
+
+	/**
+	 * Add ECE data to `wcSettings` to allow it to be accessed by the front-end JS script in the Block editor.
+	 *
+	 * @return void
+	 */
+	private function register_ece_data() {
```

Comment:
Should we rename this to `register_ece_data_for_block_editor` so it's clear that this shouldn't be loaded anywhere else?

Reply:
Renamed it to `register_ece_data_for_block_editor` in cb0772b.